.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ commands:
           key: dependency-cache-{{ checksum "package.json" }}
           paths:
             - ./node_modules
-      - run: yarn run build
+      - run: bash -c 'ci/commoncmd.sh'
```

Comment:
As I said below I don't think you need to build stuff every time, only for the mozilla job